compositions/Header/Header.css.ts

```diff
@@ -51,83 +52,60 @@ export const nounsGlasses = style([
   }),
 ])
 
-export const collectionTrigger = style([
-  atoms({
-    borderRadius: 'curved',
-    gap: 'x2',
-    px: 'x4',
-  }),
-])
-
-export const collectionTriggerWrapper = style([
+export const manageButton = style([
   {
-    gridColumn: '1',
+    gridColumn: '2',
     gridRow: '2',
+    height: 42,
+    paddingLeft: 20,
+    paddingRight: 20,
     '@media': {
       [media.min1024]: {
-        gridColumn: '3',
+        gridColumn: '23',
         gridRow: '1',
       },
     },
   },
   atoms({
-    w: '100%',
-    justifyContent: 'flex-start',
+    w: 'auto',
+    ml: 'auto',
+    borderRadius: 'curved',
+    justifyContent: {
+      '@initial': 'center',
+      '@1024': 'center',
+    },
   }),
 ])
 
-export const modalWrapper = style({
-  overflowY: 'scroll',
-  height: 400,
-})
-
-export const manageButtonWrapper = style([
+export const connectButton = style([
   {
     gridColumn: '2',
-    gridRow: '2',
+    gridRow: '1',
     '@media': {
       [media.min1024]: {
-        gridColumn: '23',
+        gridColumn: '24',
         gridRow: '1',
       },
     },
   },
   atoms({
     w: '100%',
-    justifyContent: 'flex-end',
-  }),
-])
-
-export const manageButton = style([
-  {
-    width: 130,
-  },
-  atoms({
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-    h: '100%',
-    backgroundColor: 'tertiary',
-    py: {
-      '@initial': 'x1',
-      '@1024': 'x2',
+    justifyContent: {
+      '@initial': 'flex-end',
+      '@1024': 'flex-start',
     },
   }),
 ])
 
-export const connectWrapper = style([
+export const docsLink = style([
   {
-    gridColumn: '2',
-    gridRow: '1',
-    '@media': {
-      [media.min1024]: {
-        gridColumn: '24',
-        gridRow: '1',
-      },
-    },
+    height: 42,
+    paddingLeft: space.x1,
```

Comment:
paddings and gap can go in atoms

Reply:
not when you're trying to override a button element... at least in the version currently used in the app.